apps/src/studio/skins.js

```diff
@@ -1570,31 +1602,66 @@ function loadStudio(skin, assetUrl) {
     [msg.itemRandom(), RANDOM_VALUE]];
 }
 
-
 exports.load = function (assetUrl, id) {
   var skin = skinsBase.load(assetUrl, id);
 
   // NOTE: all class names should be unique.  eventhandler naming won't work
   // if we name a projectile class 'left' for example.
   skin.ProjectileClassNames = [
+    'airplane',
+    'basketball',
+    'disc',
+    'pie',
+    'pumpkin',
+    'star',
+    'sandwich',
+    'snowball',
     'blue_fireball',
     'purple_fireball',
     'red_fireball',
     'purple_hearts',
     'red_hearts',
-    'yellow_hearts',
+    'yellow_hearts'
   ];
+
+  skin.specialProjectileProperties = {
```

Comment:
Yes, this is consumed by studio.js (https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/studio/studio.js#L4895) and lets us override this skin's default number of 8 frames per projectile for these new animations.